software/stringtie/merge/main.nf

```diff
@@ -0,0 +1,31 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process STRINGTIE_MERGE {
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['']) }
+
+    // Note: 2.7X indices incompatible with AWS iGenomes.
+    conda     (params.enable_conda ? "bioconda::stringtie=2.1.4" : null)
+    container "quay.io/biocontainers/stringtie:2.1.4--h7e0af3c_0"
```

Comment:
Didn't catch it the first time but this is missing the singularity container, should have this structure:

```
    conda (params.enable_conda ? "bioconda::bcftools=1.11" : null)
    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
        container "https://depot.galaxyproject.org/singularity/bcftools:1.11--h7c999a4_0"
    } else {
        container "quay.io/biocontainers/bcftools:1.11--h7c999a4_0"
    }
```

Reply:
I have added the singularity container for `stringtie2`